visualization/app/codeCharta/services/loadFile/fileParser.ts

```diff
@@ -52,13 +53,23 @@ export function enrichFileStatesAndRecentFilesWithValidationResults(
 
         if (fileValidationResult.errors.length === 0) {
             fileValidationResult.warnings.push(...checkWarnings(nameDataPair?.content))
-            addFile(fileStates, recentFiles, nameDataPair, currentFilesAreSampleFilesCallback, setCurrentFilesAreNotSampleFilesCallback)
+            const addedFile = addFile(
```

Comment:
small nitpick but this could be `hasAddedFile` instead of `addedFile` as its not the file itself and simply the boolean state of the operation. but to avoid this alltogether we can do
```
newFilesWereAdded = addFile(...)
```

Reply:
Good catch